### ISSUE TITLE
[Rust] Implement new Concept Exercise: tuples

### ISSUE THREAD

**Issue by efx** [exercism/v3#1302]
This issue describes how to implement the `tuples` concept exercise for the rust track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

Introduce the student to tuples and how to work with them.

## Learning objectives

- know how to create tuples 
- understand tuple access syntax `a_tuple.0`
- know how to destructure tuples into new variables, e.g. `let (x, y) = a_point;`;

## Out of scope

- tuple structs
- unit structs
- enums with tuple structs

## Concepts

- tuples
- destructuring

## Prerequisites

- structs

### Hints

- https://doc.rust-lang.org/book/ch03-02-data-types.html#the-tuple-type
- https://doc.rust-lang.org/stable/rust-by-example/primitives/tuples.html 
- https://doc.rust-lang.org/stable/rust-by-example/flow_control/match/destructuring.html

### After

- https://doc.rust-lang.org/book/ch03-02-data-types.html#the-tuple-type
- https://doc.rust-lang.org/stable/rust-by-example/primitives/tuples.html 
- https://doc.rust-lang.org/stable/rust-by-example/flow_control/match/destructuring.html

## Representer

No changes required

## Analyzer

No changes required

## Implementing
See our [implementation guide](https://github.com/exercism/v3/blob/master/languages/rust/howto/implementing-a-concept-exercise.md#how-to-implement-a-rust-concept-exercise) for guidance. In addition you may want to make use of the shell script while in the `concept` directory, with the arg loops, e.g. `sh boil_exercise.sh tuples`

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by efx** [comment]
The current rust track does not [explicitly](https://github.com/exercism/rust/blob/master/config.json) introduce this concept. 

Thus, we should keep the tuples concept exercise as simple as possible. Riffing of of RBE, I think [the following](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=87170cd31e74974738243cfb54041b79) will work.
